.gitignore

```diff
@@ -4,4 +4,8 @@ node_modules/
 dist/
 node_modules
 /lib
-dist
\ No newline at end of file
+dist
+.enyocache
```

Comment:
We should only ignore .enyocache, and not ignore appinfo, icon, or largeicon. Those are used when compiling strawman into a packaged app.